packages/calendar/core/src/helpers/locale.ts

```diff
@@ -1,4 +1,4 @@
-import { Language } from "../types";
+import { Language } from "@agreeto/api/types";
```

Comment:
Will this contain more types than just models?

It may, right? e.g. we could include the google calendar api types in there too and distribute this package to the apps that depend on it?

Reply:
ideally most types will be inferred through trpc.

the prisma enums are a bit of a special case imo